app/resonant-laboratory/ansible/xenial/girder.service.template

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Girder
+After=network.target
+
+[Service]
```

Comment:
I'll need to confer with IT before I make this change, so here's an issue describing the problem: #390